### PR TITLE
Remove option to read from standard input in `fit_speck_removal`

### PR DESCRIPTION
**Pull request by ecbland**
This PR "fixes" #424 by simply removing the possibility of reading from `stdin`. The problem is that `fit_speck_removal` has to read the file twice in order to perform the filtering, and `rewind(fp)` does not work if `fp=stdin`. 

On `develop`, we get the following error when `fit_speck_removal` reads from `stdin`:
```
make_fit 20210529.2200.00.cly.a.rawacf | fit_speck_removal > out.fitacf
Error reading file
free(): invalid pointer
Aborted (core dumped)
```

On this branch, there will be a `File not found` error instead. 

_Note: There are a few other routines in RST that use `rewind(fp)` and therefore behave slightly differently depending on whether `fp` is a file or `stdin`. I'll make a new bug report to capture this._

### CODE REVIEW

**Comment by mts299** [comment]
Personally I am not a fan of "file not found" error as it makes no sense and not descriptive but the PR works as expected.

File: codebase/superdarn/src.bin/tk/tool/fit_speck_removal.1.0/fit_speck_removal.c [hunk 1]
```diff
@@ -134,10 +134,7 @@ int main (int argc,char *argv[]) {
   }
   
   FILE *fp=NULL;
-  if (arg==argc) 
-      fp=stdin;
-  else 
-      fp=fopen(argv[arg],"r");
+  fp=fopen(argv[arg],"r");
   if (fp==NULL) {
     fprintf(stderr,"File not found.\n");
```
Comment (ecbland) [inline codebase/superdarn/src.bin/tk/tool/fit_speck_removal.1.0/fit_speck_removal.c:139]: @mts299 Is this better than "file not found"?
```suggestion
    fprintf(stderr,"Error opening input file.\n");
```

Comment (mts299) [reply codebase/superdarn/src.bin/tk/tool/fit_speck_removal.1.0/fit_speck_removal.c:139]: How about that? ^ You can commit if you like.

Comment (mts299) [reply codebase/superdarn/src.bin/tk/tool/fit_speck_removal.1.0/fit_speck_removal.c:139]: Can you seperate the two? file not found makes sense for the later

**Pull request merged**